TaisKohtApi/Controllers/api/IngredientsController.cs

```diff
@@ -114,14 +114,14 @@ public IActionResult Post([FromBody]PostIngredientDTO ingredientDTO)
         ///     }
         ///
         /// </remarks>
-        /// <response code="204">Ingredient was successfully updated, no content to be returned</response>
+        /// <response code="200">Ingredient was successfully updated, updated Ingredient to be returned</response>
         /// <response code="400">Faulty request, please review ID and content body</response>
         /// <response code="429">Too many requests</response>
         /// <response code="500">Internal error, unable to process request</response>
         // PUT: api/v1/Ingredients/5
         [Authorize(Roles = "admin, normalUser, premiumUser")]
         [HttpPut("{id}")]
-        [ProducesResponseType(204)]
+        [ProducesResponseType(typeof(IngredientDTO), 200)]
```

Comment:
PUT ei tagasta 200, normaalne response on 204 (operation completed successfully), kui tahta tagastada objekti, võiks olla 201 at the very least. 
**https://stackoverflow.com/questions/797834/should-a-restful-put-operation-return-something?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa**

Reply:
https://httpstatuses.com/204 -> no content aga me tagastame uue objekti